internal/plans/action.go

```diff
@@ -16,6 +16,9 @@ const (
 	CreateThenDelete Action = '±'
 	Delete           Action = '-'
 	Forget           Action = '.'
+	Open             Action = '⁐'
+	Renew            Action = '◌'
```

Comment:
I spoke with @Yantrio earlier today and we discussed the hooks.  In practice, the only one of these "actions" used outside of Hooks is Open (which is well documented below).

Do we want to get rid of Renew and Close here and instead create Ephemeral specific hooks?  There is some precedent for that today with the non-resource hooks.

Reply:
Removed the Renew and Close action and added specific hook methods for those in 008c9521abc1234db80c728dde71d59ddcb4c9bf